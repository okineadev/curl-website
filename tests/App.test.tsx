import { expect, test } from 'bun:test'

import { render } from 'ink-testing-library'

// @ts-ignore
import App from '../src/App.tsx'

test('just works', () => {
	const { lastFrame } = render(<App />)

	expect(lastFrame()).toMatchSnapshot()
})
