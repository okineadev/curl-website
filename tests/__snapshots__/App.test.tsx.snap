// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`just works 1`] = `
"\x1B[90m╭─────────────────────────────────────────────────────────────────────────────────────────────╮\x1B[39m
\x1B[90m│\x1B[39m \x1B[31m●\x1B[39m \x1B[33m●\x1B[39m \x1B[32m●\x1B[39m   ⯇   \x1B[2m⯈\x1B[22m                \x1B[2m【\x1B[22m         \x1B[97m\x1B]8;;https://okinea.dev\x07okinea.dev\x1B[39m\x1B]8;;\x07      ↻  \x1B[2m】\x1B[22m\x1B]8;;\x07                             \x1B[1m+\x1B[22m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[90m───────────────────────────────────────────────────────────────────────────────────────────\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m Hello 👋, my name is \x1B[1mYura\x1B[22m, I'm 16 years old, programmer, designer, I create cool websites,  \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m amazing CLI applications and I also write posts on Medium from time to time :)              \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                                                                             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m I work on open source projects, create my own and help others. I am the (ex) maintainer of  \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m the \x1B]8;;https://github.com/material-extensions/vscode-material-icon-theme\x07\x1B[1mMaterial Icon Theme\x1B[22m\x1B]8;;\x07 extension for VS Code, Co-founder of \x1B]8;;https://github.com/ZGalera\x07\x1B[1mZgalera Inc\x1B[22m\x1B]8;;\x07.\x1B]8;;\x07                   \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                                                                             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                         \x1B[1m📁 Projects\x1B[22m                                         \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                                                                             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m      \x1B]8;;https://github.com/okineadev/material-icons-figma-plugin\x07\x1B[32mmaterial-icons-figma-plugin\x1B[39m\x1B]8;;\x07 \x1B]8;;https://github.com/okineadev/cli-tips\x07\x1B[32mcli-tips\x1B[39m\x1B]8;;\x07 \x1B]8;;https://github.com/just-serverless/contributors-table\x07\x1B[32mcontributors-table\x1B[39m\x1B]8;;\x07 \x1B]8;;https://github.com/okineadev/okineadev-website\x07\x1B[32mokineadev-website\x1B[39m\x1B]8;;\x07 \x1B]8;;https://github.com/okineadev/dotload\x07\x1B[32mdotload\x1B[39m\x1B]8;;\x07      \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                                                                             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                \x1B]8;;https://github.com/okineadev/platform-tools-installer-gui\x07\x1B[32mplatform-tools-installer-gui\x1B[39m\x1B]8;;\x07 \x1B]8;;https://github.com/cli-stuff/platform-tools-installer-windows\x07\x1B[32mplatform-tools-installer-windows\x1B[39m\x1B]8;;\x07                \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                                                                             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                         \x1B[1m🌍 Socials\x1B[22m                                          \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                                                                             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m             \x1B]8;;https://github.com/okineadev\x07\x1B[31mGitHub\x1B[39m\x1B]8;;\x07 \x1B]8;;https://unsplash.com/@okineadev\x07\x1B[31mUnsplash\x1B[39m\x1B]8;;\x07 \x1B]8;;https://telegram.okinea.dev\x07\x1B[31mTelegram\x1B[39m\x1B]8;;\x07 \x1B]8;;mailto:hi@okinea.dev\x07\x1B[31mEmail\x1B[39m\x1B]8;;\x07 \x1B]8;;https://okineadev.medium.com/\x07\x1B[31mMedium\x1B[39m\x1B]8;;\x07 \x1B]8;;https://dev.to/okineadev\x07\x1B[31mDEV.to\x1B[39m\x1B]8;;\x07 \x1B]8;;https://discord.com/users/okineadev\x07\x1B[31mDiscord\x1B[39m\x1B]8;;\x07 \x1B]8;;https://stackoverflow.com/users/21165921/okinea-dev\x07\x1B[31mStack Overflow\x1B[39m\x1B]8;;\x07             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                                                                             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                   \x1B[1m📰 My posts on \x1B[102m\x1B[30m Medium \x1B[39m\x1B[49m\x1B[22m                                   \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m                                                                                             \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m╭───────────────────────────────────────────╮\x1B[39m \x1B[36m╭───────────────────────────────────────────╮\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m│\x1B[39m                                           \x1B[36m│\x1B[39m \x1B[36m│\x1B[39m                                           \x1B[36m│\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m│\x1B[39m \x1B]8;;https://medium.com/@okineadev/how-to-get-a-free-macos-machine-for-testing-b2f6c72415fd\x07\x1B[1m\x1B[34mHow to get a free macOS machine for \x1B[39m\x1B[22m\x1B]8;;\x07      \x1B[36m│\x1B[39m \x1B[36m│\x1B[39m \x1B]8;;https://medium.com/@okineadev/how-to-set-up-vnc-on-a-headless-raspberry-pi-with-ubuntu-server-24-04-96cc8ddf65bc\x07\x1B[1m\x1B[34mHow to set up VNC on a headless Raspberry\x1B[39m\x1B[22m\x1B]8;;\x07 \x1B[36m│\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m│\x1B[39m \x1B]8;;https://medium.com/@okineadev/how-to-get-a-free-macos-machine-for-testing-b2f6c72415fd\x07\x1B[34mtesting?\x1B[39m\x1B]8;;\x07                                  \x1B[36m│\x1B[39m \x1B[36m│\x1B[39m \x1B]8;;https://medium.com/@okineadev/how-to-set-up-vnc-on-a-headless-raspberry-pi-with-ubuntu-server-24-04-96cc8ddf65bc\x07\x1B[34m Pi...\x1B[39m\x1B]8;;\x07                                    \x1B[36m│\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m│\x1B[39m \x1B[2mif you have created a program and want to\x1B[22m \x1B[36m│\x1B[39m \x1B[36m│\x1B[39m \x1B[2mIf you have Ubuntu Server installed on \x1B[22m   \x1B[36m│\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m│\x1B[39m \x1B[2m test it on different operating systems \x1B[22m  \x1B[36m│\x1B[39m \x1B[36m│\x1B[39m \x1B[2myour Raspberry Pi and you want to connect\x1B[22m \x1B[36m│\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m│\x1B[39m \x1B[2m(including macOS), but if...\x1B[22m              \x1B[36m│\x1B[39m \x1B[36m│\x1B[39m \x1B[2m to its interface without a monitor or \x1B[22m   \x1B[36m│\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m│\x1B[39m                                           \x1B[36m│\x1B[39m \x1B[36m│\x1B[39m \x1B[2mmouse?\x1B[22m                                    \x1B[36m│\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m│\x1B[39m                                           \x1B[36m│\x1B[39m \x1B[36m│\x1B[39m                                           \x1B[36m│\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m│\x1B[39m \x1B[36m╰───────────────────────────────────────────╯\x1B[39m \x1B[36m╰───────────────────────────────────────────╯\x1B[39m \x1B[90m│\x1B[39m
\x1B[90m╰─────────────────────────────────────────────────────────────────────────────────────────────╯\x1B[39m"
`;
